securitycenter/src/main/java/com/google/cloud/examples/securitycenter/snippets/NotificationConfigSnippets.java

```diff
@@ -0,0 +1,161 @@
+      throws IOException {
+
+    // [START scc_delete_notification_config]
+    // String organizationId = "{your-org-id}";
+    // String notificationConfigId = "config-id";
+
+    NotificationConfigName notificationConfigName =
+        NotificationConfigName.newBuilder()
+            .setOrganization(organizationId)
+            .setNotificationConfig(notificationConfigId)
+            .build();
+
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      client.deleteNotificationConfig(notificationConfigName);
+
+      System.out.println(String.format("Deleted Notification config: %s", notificationConfigName));
+    }
+    // [END scc_delete_notification_config]
+
+    return true;
+  }
+
+  public static NotificationConfig getNotificationConfig(
+      String organizationId, String notificationConfigId) throws IOException {
+
+    // [START scc_get_notification_config]
+    // String organizationId = "{your-org-id}";
+    // String notificationConfigId = "config-id";
+
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      NotificationConfig response =
+          client.getNotificationConfig(
+              NotificationConfigName.newBuilder()
+                  .setOrganization(organizationId)
+                  .setNotificationConfig(notificationConfigId)
+                  .build());
+
+      System.out.println(String.format("Notification config: %s", response));
+      return response;
+    }
+    // [END scc_get_notification_config]
+  }
+
+  public static ImmutableList<NotificationConfig> listNotificationConfigs(String organizationId)
+      throws IOException {
+    // [START scc_list_notification_configs]
+    // String organizationId = "{your-org-id}";
+
+    OrganizationName orgName =
+        OrganizationName.newBuilder().setOrganization(organizationId).build();
+
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      ListNotificationConfigsPagedResponse response = client.listNotificationConfigs(orgName);
+
+      ImmutableList<NotificationConfig> notificationConfigs =
+          ImmutableList.copyOf(response.iterateAll());
+      System.out.println(
+          String.format("List notifications response: %s", response.getPage().getValues()));
+      return notificationConfigs;
+    }
+    // [END scc_list_notification_configs]
+  }
+
+  public static NotificationConfig updateNotificationConfig(
+      String organizationId, String notificationConfigId, String pubsubTopic) throws IOException {
+    // [START scc_update_notification_config]
+    // String organizationId = "{your-org-id}";
+    // String notificationConfigId = "{your-config-id}";
+    String notificationConfigName =
+        String.format(
+            "organizations/%s/notificationConfigs/%s", organizationId, notificationConfigId);
+
+    // String pubsubTopic = "projects/{your-project}/topics/{your-new-topic}";
```

Comment:
We should format things for them so they just provide the projectID and topic name.